www/m/index.html

```diff
@@ -244,34 +240,8 @@ <h2 class="mdl-card__title-text">Fortunes</h2>
       <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 500 250" style="position: fixed; left: -1000px; height: -1000px;">
         <defs>
           <g id="chart">
-            <g id="Gridlines">
```

Comment:
Done :)
But this template is not used by the entropy chart anymore. The entropy chart is inline svg.